### ISSUE TITLE
[question] is there a login page for parse dashboard?

### ISSUE THREAD

**Issue by okonon** [ParsePlatform/parse-dashboard#516]
[question] is there a login page for parse dashboard?

**Comment by dvanwinkle** [comment]
No, it uses [Basic Authentication](https://github.com/ParsePlatform/parse-dashboard#configuring-basic-authentication)

**Comment by okonon** [comment]
I was wondering if anyone has any ideas how to secure access to the dashboard

**Comment by flovilmart** [comment]
You can secure it with basic auth, over https it isn't that bad. If we wanna go further, we':d need a clientside login page as well as a server endpoint to exchange the username/pass with an expirable session toeken. This session token could be passed along each request as basic auth header. If the session is expired, the server should redirect to the login page.
